Cubical/Algebra/Group/Free.agda

```diff
@@ -0,0 +1,487 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Algebra.Group.Free where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Structure
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Data.Bool as 𝟚 hiding (_≤_)
+open import Cubical.Data.Nat as ℕ hiding (_·_)
+open import Cubical.Data.Nat.Order.Recursive as OR
+open import Cubical.Data.Empty as ⊥
+open import Cubical.Data.Sum as ⊎
+open import Cubical.Data.Sigma
+open import Cubical.Data.List as Li
+
+open import Cubical.Relation.Nullary
+open import Cubical.Relation.Binary
+
+open import Cubical.Functions.Involution
+
+open import Cubical.Algebra.Group
+
+open import Cubical.HITs.SetQuotients as SQ renaming (_/_ to _/₂_ ; [_] to [_]/)
+
+
+
+private
+  variable
+    ℓ : Level
+
+[𝟚×_] : (A : Type ℓ) → Type ℓ
+[𝟚×_] = List ∘ (Bool ×_)
```

Comment:
Here could be a comment which explains that the ```Bool``` will be used for inverses.

Reply:
added note in overview at the beggining of the file